docs/vsts-README.md

```diff
@@ -1,45 +1,45 @@
-# Visual Studio Team Services Extension for the App Store
+# Visual Studio Team Services Extension for the Apple App Store
 
 This extension contains a set of deployment tasks which allow you to automate the release and promotion of app updates to Apple's App Store from your CI environment. This can reduce the effort needed to keep your beta and production deployments up-to-date, since you can simply push changes to the configured source control branches, and let your automated build take care of the rest.
 
 ## Prerequisites
 
 * In order to automate the release of app updates to the App Store, you need to have manually released at least one version of the app beforehand.
-* The tasks install and use [fastlane](https://github.com/fastlane/fastlane) tools. Fastlane requires Ruby 2.0.0 or above and recommends having the latest Xcode command line tools installed on the MacOS computer. 
+* The tasks install and use [fastlane](https://github.com/fastlane/fastlane) tools. fastlane requires Ruby 2.0.0 or above and recommends having the latest Xcode command line tools installed on the MacOS computer. 
```

Comment:
Should you link to their product page (https://fastlane.tools/) instead of their Github page? The product page has a GitHub link front and center.

Reply:
Either one is fine in my opinion since docs point to GitHub and GitHub readme points to docs :)